code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -25,9 +25,15 @@
 	result_amount = 4
 	required_temp = 424
 
+datum/reagent/clf3/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustFireLoss(1*REM)
```

Comment:
CFL3's danger is largely in its ability to set off smoke powder and black powder (both not in yet) instantly), which is why it's not particularly "exciting" in terms of a per cycle effect.

On Goon it just gives you a lot of fire stacks and ignites you every life proc its' in your system; if you inhale it while temperature sensitive reagents are in your system? well, it'll cause those to react almost instantaneously (which is why it's so good despite not doing any direct fire damage).